web/docs/installation/deploy-the-platform.md

```diff
@@ -128,67 +128,111 @@ The platform uses four URLs that require a HTTP reverse proxy. These URLs may be
 mapped to the same or different hostnames.
 
 1. The URL that the user's browser connects to, e.g.,
-   `app.platform.example.org`. This serves a web frontend where the user can
+   `app.platform.example`. This serves a web frontend where the user can
    interact with the platform.
-2. The URL that the nodes connect to, e.g., `nodes.platform.example.org`. Tenzir
+2. The URL that the nodes connect to, e.g., `nodes.platform.example`. Tenzir
    Nodes connect to this URL to establish long-running WebSocket connections.
 3. The URL that the platform's S3-compatible blob storage is accessible at,
-   e.g., `downloads.platform.example.org`. When using the *Download* button
+   e.g., `downloads.platform.example`. When using the *Download* button
    the platform generates download links under this URL.
 4. The URL that the Tenzir Platform CLI connects to, e.g.,
-   `api.platform.example.org`.
+   `api.platform.example`.
 
 You must provide the following environment variables to the platform:
 
 ```bash
 # The domain under which the platform frontend is reachable. Must include the
 # `http://` or `https://` scheme.
-TENZIR_PLATFORM_DOMAIN=https://app.platform.example.org
+TENZIR_PLATFORM_DOMAIN=https://app.platform.example
 
 # The endpoint to which Tenzir nodes should connect. Must include the `ws://`
 # or `wss://` scheme.
-TENZIR_PLATFORM_CONTROL_ENDPOINT=wss://nodes.platform.example.org
+TENZIR_PLATFORM_CONTROL_ENDPOINT=wss://nodes.platform.example
 
 # The URL at which the platform's S3-compatible blob storage is accessible at.
-TENZIR_PLATFORM_BLOBS_ENDPOINT=https://downloads.platform.example.org
+TENZIR_PLATFORM_BLOBS_ENDPOINT=https://downloads.platform.example
 
 # The URL at which the platform's S3-compatible blob storage is accessible at.
-TENZIR_PLATFORM_API_ENDPOINT=https://api.platform.example.org
+TENZIR_PLATFORM_API_ENDPOINT=https://api.platform.example
 ```
 
 ### Identity Provider (IdP)
 
-The platform requires an external Identity Provider (IdP) supporting the OIDC
-protocol. The IdP must provide valid RS256 ID tokens. The platform must be able
-to access the IdP's issuer URL.
+The Tenzir Platform can be configured to use an external Identity Provider.
+This is required when using the platform with multiple users.
```

Comment:
How does "multiple users" relate with "external IdP"?

Reply:
Without an external IdP, you can only use the single-user mode of the platform. (aka the `localdev` mode) Not sure how to formulate this so it makes sense.

Reply:
How about simply:

> You can configure the Tenzir Platform with an external Identity Provider.

I think here it doesn't matter what the internal IdP is. You could add:

> , instead of using dex, which we use in our Docker Compose examples.

Reply:
Dex is also an external IdP for the purposes of this document (and we're not using it anymore in any of our examples), but just leaving out the second sentence completely sounds like a good solution to me.